The included tests are simple, "hello world" style tests
that do not actually test this package.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -187,7 +187,3 @@
                             /dVcInstallDir=$ENV{VCINSTALLDIR}
                        )
 ENDIF (WIN32)
-
-# Unit-Tests
-ENABLE_TESTING()
-ADD_SUBDIRECTORY(tests)
